### PR TITLE
chore (docs): proposal for a uniform update of "Model Capabilities" section

### PR DESCRIPTION
**Pull request by ggallon**
Hi,

I noticed that the “Model Capabilities” section was becoming less uniform.
I propose an update of this section for all providers.

What do you think?

### CODE REVIEW

File: content/providers/01-ai-sdk-providers/20-mistral.mdx [hunk 1]
```diff
@@ -111,19 +111,20 @@ Mistral language models can also be used in the `streamText`, `generateObject`,
...
+
 | Model                  | Image Input         | Object Generation   | Tool Usage          | Tool Streaming      |
 | ---------------------- | ------------------- | ------------------- | ------------------- | ------------------- |
-| `pixtral-large-latest` | <Check size={18} /> | <Check size={18} /> | <Check size={18} /> | <Check size={18} /> |
 | `mistral-large-latest` | <Cross size={18} /> | <Check size={18} /> | <Check size={18} /> | <Check size={18} /> |
 | `mistral-small-latest` | <Cross size={18} /> | <Check size={18} /> | <Check size={18} /> | <Check size={18} /> |
-| `ministral-3b-latest`  | <Cross size={18} /> | <Check size={18} /> | <Check size={18} /> | <Check size={18} /> |
 | `ministral-8b-latest`  | <Cross size={18} /> | <Check size={18} /> | <Check size={18} /> | <Check size={18} /> |
+| `ministral-3b-latest`  | <Cross size={18} /> | <Check size={18} /> | <Check size={18} /> | <Check size={18} /> |
+| `pixtral-large-latest` | <Check size={18} /> | <Check size={18} /> | <Check size={18} /> | <Check size={18} /> |
```
Comment (lgrammel) [inline content/providers/01-ai-sdk-providers/20-mistral.mdx:122]: Please don't re-order. The order should also reflect model importance.

File: content/providers/01-ai-sdk-providers/20-mistral.mdx [hunk 2]
```diff
@@ -111,19 +111,20 @@ Mistral language models can also be used in the `streamText`, `generateObject`,
...
 | `pixtral-12b-2409`     | <Check size={18} /> | <Check size={18} /> | <Check size={18} /> | <Check size={18} /> |
 
 <Note>
-  The table above lists popular models. Please see the [Mistral
-  docs](https://docs.mistral.ai/getting-started/models/models_overview/) for a
-  full list of available models.
```
Comment (lgrammel) [inline content/providers/01-ai-sdk-providers/20-mistral.mdx:126]: This is the format that I would want for all model providers.

**Comment by lgrammel** [comment]
Thanks for looking into this! The format I would want is a single note per provider, with a descriptive link (instead of 'here'). The original Mistral version is a good template.

**Comment by ggallon** [comment]
@lgrammel, to be sure of your return, it's this layout?

### Model Capabilities

~~The latest Mistral models list is available [here](https://docs.mistral.ai/getting-started/models/models_overview/) documentation for up to date information, including:~~

| Model                  | Image Input         | Object Generation   | Tool Usage          | Tool Streaming      |
| ---------------------- | ------------------- | ------------------- | ------------------- | ------------------- |
| `pixtral-large-latest` | <Check size={18} /> | <Check size={18} /> | <Check size={18} /> | <Check size={18} /> |
| `mistral-large-latest` | <Cross size={18} /> | <Check size={18} /> | <Check size={18} /> | <Check size={18} /> |
| `...` | <Cross size={18} /> | <Check size={18} /> | <Check size={18} /> | <Check size={18} /> |

`<Note>
    The table above lists popular models. Please see the [Mistral docs](https://docs.mistral.ai/getting-started/models/models_overview/) for a full list of available models.
    You can also pass any available provider model ID as a string if needed.
</Note>`

**Comment by lgrammel** [comment]
@ggallon yes

**Review by lgrammel — APPROVED** [review]

**Pull request merged**